### ISSUE TITLE
filesize is not a function

### ISSUE THREAD

**Issue by DrimZ** [amitdahan/ngx-filesize#385]
Already resolved this but just incase someone encounter the same error.

since the "filesize" dependency was declared like this:
`"dependencies": {
    "filesize": ">= 4.0.0",
}`

it will always install the latest version of `filesize`, which leads to this error due to the recent release of filesize v10, and it seems there are some breaking changes in there.

to resolve this we need to downgrade the filesize version to v9, to do this you need to add this piece of code in your `package.json` file

```
"dependencies": {
   // other dependencies
   "ngx-filesize": "^2.0.16",
},
"overrides": {
    "ngx-filesize": {
        "filesize": "9.0.11"
    }
}
```

then run `npm install`  again.

note: `overrides` is only available for npm v8 and above

**Comment by ivankenig** [comment]
@DrimZ "overrides" did not work for me. I don't know why (npm 8.11.0, Angular CLI: 14.2.5, Node: 16.16.0, yarn 1.22.19)

Solved the issue with "resolutions":
`"dependencies": {
   // other dependencies
   "ngx-filesize": "^2.0.16",
},
"resolutions": {
    "filesize": "9.0.11"
}`

**Comment by DrimZ** [comment]
cool i think "overrides" is only supported by `npm`, but if you are using `yarn` you will then use "resolutions"

**Comment by kleinph** [comment]
Will this be fixed? Is this project maintained anymore (`resolutions` or `overrides` don't seem to work on stackblitz, where I used this lib in a showcase.)

**Comment by amitdahan** [comment]
Hey, I released `ngx-filesize@3.0.0-alpha.0` with `filesize` as a `peerDependency` and bumped the Angular major requirement & support to 14.

I'll close this issue once `3.0.0` is out.
